source/unit/Fixture.ooc

```diff
@@ -170,9 +170,10 @@ Fixture: abstract class {
 }
 
 TestFailedException: class extends Exception {
+	description: String
```

Comment:
The `CharBuffer` constructor takes a `stringLiteral` parameter, just saving it would be even easier.